src/SFProvider.cc

```diff
@@ -1,89 +1,97 @@
 #include "TauAnalysisTools/TauTriggerSFs/interface/SFProvider.h"
 
+
 #include <algorithm>
 #include <sstream>
 
 namespace tau_trigger {
 
-const std::set<int> SFProvider::supported_decay_modes = { 0, 1, 10, 11 };
-
+const std::set<int> SFProvider::supported_decay_modes = { 0, 1, 10, 11};
+  
 SFProvider::SFProvider(std::string_view input_file, std::string_view channel, std::string_view wp)
 {
     TFile root_file(input_file.data(), "READ");
     if(root_file.IsZombie()) {
-        std::ostringstream ss;
-        ss << "tau_trigger::SFProvider: unable to open \"" << input_file << "\".";
-        throw std::runtime_error(ss.str());
+      std::ostringstream ss;
+      ss << "tau_trigger::SFProvider: unable to open \"" << input_file << "\".";
+      throw std::runtime_error(ss.str());
     }
-
+    
     std::map<std::string, std::map<int, std::unique_ptr<TH1F>>*> histograms = {
-        { "data", &eff_data }, { "mc", &eff_mc }, { "sf", &sf }
+      { "data", &eff_data }, { "mc", &eff_mc }, { "sf", &sf }
     };
+            
     for(int dm : supported_decay_modes) {
-        for(const auto& entry : histograms) {
-            std::ostringstream ss_hist_name;
-            // For the vbf ditau trigger, the efficiencies of the 3 prong decay modes are merged.
-            if (channel == "ditauvbf" && (dm == 10 || dm == 11))
-            {
-                ss_hist_name << entry.first << "_" << channel << "_" << wp << "_dm1011_fitted";
-            }
-            else
-            {
-                ss_hist_name << entry.first << "_" << channel << "_" << wp << "_dm" << dm << "_fitted";
-            }
-            (*entry.second)[dm].reset(LoadHistogram(root_file, ss_hist_name.str()));
-        }
+      for(const auto& entry : histograms) {
+	std::  ostringstream ss_hist_name;
```

Comment:
Please add more indentation to this line such that it is indented four whitespaces from the for clause and remove the whitespaces after `::`.